client/src/components/material.jsx

```diff
@@ -139,12 +138,25 @@ class Material extends React.Component {
 
   render() {
     const materials = this.props.materials;
+    if(!materials.length) {
+      return <Carousel>
+          <Card key='none' onMouseEnter={this.handleHover} onMouseLeave={this.handleHover} style={{backgroundColor: "#f5d90a"}}>
+          {this.state.isHovered ? <MatInfo style={{color: "black", fontSize: "2rem"}}>Let’s quack this case 🐣.</MatInfo> : null }
+          <Blob url="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcT2EGNOn0R__P2Bv_oIzb2EIRAQCAYKO7xQaw&usqp=CAU"></Blob>
+          <MatName>100% vegan, range free rubber duck</MatName>
+          <MatDesc>What did the duck detective say to his partner?</MatDesc>
+          <Program>Program</Program>
+          <LearnMore>Learn More</LearnMore>
+        </Card>
+      </Carousel>
+    }
+
     const { isHovered } = this.state;
```

Comment:
what if you move line 154 to underneath line 140? That way the destructuring you are doing here would apply to everything below it, including inside the edge case on line 144 where you are doing the if/else statement?

Reply:
Ah! totally didn't see this. Done!